### ISSUE TITLE
Cannot find 'embedManifest' in class com.github.maven_nar.NarCompileMojo

### ISSUE THREAD

**Issue by nicoulaj** [sdedwards/m2e-nar#3]
I'm trying to use this connector, but I get:
```
Unable to parse configuration of mojo com.github.maven-nar:nar-maven-plugin:3.5.1:nar-compile for parameter embedManifest: Cannot find 'embedManifest' in class com.github.maven_nar.NarCompileMojo
Cannot find 'embedManifest' in class com.github.maven_nar.NarCompileMojo
```
I tried with various versions of NAR plugin on my project including 3.5.2, I always get the same error.

m2e-nar was compiled from this repository and installed manually as I cannot find an update site or installation instructions anywhere.

**Comment by sdedwards** [comment]
The connector needs updating for the newer versions of the NAR plugin. I'll try to do it within the week. In the mean time you can try removing `embedManifest` from your pom file.

The update site archives can be downloaded on the [releases page](https://github.com/maven-nar/m2e-nar/releases). I'll update the readme to make this clear.

**Comment by nicoulaj** [comment]
FYI, My POM does not contain `embedManifest`, and I tried downgrading nar-maven-plugin down to 3.3.0, but I get the same error.

Thanks for the link, I missed it!

**Comment by sdedwards** [comment]
Ok thanks for trying. It's worse than I thought then. I'll take a look.

On 12 Jan 2018 18:30, "nicoulaj" <notifications@github.com> wrote:

FYI, My POM does not contain embedManifest, and I tried downgrading
nar-maven-plugin down to 3.3.0, but I get the same error.

Thanks for the link, I missed it!

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/maven-nar/m2e-nar/issues/3#issuecomment-357299665>, or mute
the thread

**Comment by Jebonicus** [comment]
I've managed to get my eclipse project working which uses nar-maven-plugin:3.5.2. I had to add the embedManifest property to the m2e-nar plugin, along with several other properties.

See my fork for the complete changes I've made:
https://github.com/Jebonicus/m2e-nar/commit/43f4f0010611d220caecb39e3bc68cbc296b31e1

This is not a "proper" fix - I guess to do things properly most of the updated classes from the nar-maven-plugin (e.g. NarCompileMojo) would need their updates mirroring in the 'duplicate' classes in this plugin. I'm not familiar enough with the codebase, or eclipse plugin development, to attempt this, so here is my sticking plaster :)
